docs/DeveloperGuide.md

```diff
@@ -274,29 +269,40 @@ _{Explain here how the data archiving feature will be implemented}_
+|----------|--------------------------|-----------------------------------------------------------|------------------------------------------------------------|
+| `* * *`  | user                     | add new customers with details such as name and address   | remember details of customers for order deliveries         |
+| `* * *`  | user                     | add orders made by a customer                             | keep track of orders made by each customer                 |
+| `* * *`  | user                     | add details to orders, such as delivery date and quantity | remember details of orders when marking deliveries         |
+| `* * *`  | user                     | mark orders as completed                                  | keep track of orders that have been delivered              |
+| `* * *`  | user with many customers | delete a customer from my address book                    | remove clients who I no longer need to be in contact with  |
+| `* * *`  | user with many customers | view all my customer contacts                             | see an overview of all my customers' details               |
+| `* * *`  | user with many orders    | view all orders under a specific customer                 | see an overview of all orders made by a customer           |
+| `* * *`  | user with many orders    | delete an order under a customer                          | remove orders that I no longer need to track               |
+| `* * *`  | user with many orders    | revert an order's completed status                        | keep track of erroneous or failed order deliveries         |
+| `* *`    | tech-savvy user          | save data to local storage                                | keep my data even after exiting SellSavvy                  |
+| `* *`    | tech-savvy user          | load data from local storage                              | access my local data using SellSavvy                       |
+| `* *`    | experienced user         | edit a customer's details                                 | keep the customers' information up-to-date                 |
+| `* *`    | experienced user         | edit an order's details                                   | keep the orders' information up-to-date                    |
+| `*`      | experienced user         | find a customer by name                                   | search for a specific customer's details                   |
+| `*`      | experienced user         | filter orders by their status                             | see which orders are completed or have yet to be delivered |
+| `*`      | inexperienced user       | be informed a customer already made an identical order    | take note of duplicate orders made by the same customer    |
+| `*`      | inexperienced user       | be informed if a new order's delivery date has passed     | take note of erroneous creation of historical orders       |
```

Comment:
what about the view help? i rmb it is in our documents

Reply:
not sure if its that important so i didn't add.

Reply:
ok